design-tokens/tokens/layer1/base.json

```diff
@@ -207,12 +207,42 @@
       }
     },
     "lineHeight": {
-      "heading": {
-        "value": "132%",
+      "10": {
+        "value": "1.125rem",
         "type": "lineHeights"
       },
-      "body": {
-        "value": "142%",
+      "20": {
+        "value": "1.25rem",
+        "type": "lineHeights"
+      },
+      "30": {
+        "value": "1.5rem",
+        "type": "lineHeights"
+      },
+      "40": {
+        "value": "1.875rem",
+        "type": "lineHeights"
+      },
+      "50": {
+        "value": "2.25rem",
+        "type": "lineHeights"
+      },
+      "60": {
+        "value": "3.125rem",
+        "type": "lineHeights"
+      },
+      "lineHeight": {
```

Comment:
This shouldn't be nested, because of that the token name is `...lineHeight.lineHeight.heading`

Reply:
Yeah, I will do that after we safely remove the old group.

Reply:
so if I understand correctly you want to keep those value for backward compatibility? If that's the case then you should not nest them otherwise the backward compatibility is broken.
So either un-nest them or drop them.

Reply:
I dropped the values and updated the typography composition with new ones. It shouldn't be a problem now